protocol/synthetix/contracts/interfaces/IMarketManagerModule.sol

```diff
@@ -116,25 +109,8 @@ interface IMarketManagerModule {
      * @param marketId The id of the market from which snxUSD will be withdrawn.
      * @param target The address of the account that will receive the withdrawn snxUSD.
      * @param amount The amount of snxUSD to be withdraw, denominated with 18 decimals of precision.
-     * @return feeAmount the amount of fees paid (billed as additional debt towards liquidity providers)
      */
-    function withdrawMarketUsd(
-        uint128 marketId,
-        address target,
-        uint256 amount
-    ) external returns (uint256 feeAmount);
-
-    /**
-     * @notice Get the amount of fees paid in USD for a call to `depositMarketUsd` and `withdrawMarketUsd` for the given market and amount
-     * @param marketId The market to check fees for
-     * @param amount The amount deposited or withdrawn in USD
-     * @return depositFeeAmount the amount of USD paid for a call to `depositMarketUsd`
-     * @return withdrawFeeAmount the amount of USD paid for a call to `withdrawMarketUsd`
-     */
-    function getMarketFees(
-        uint128 marketId,
-        uint256 amount
-    ) external view returns (uint256 depositFeeAmount, uint256 withdrawFeeAmount);
+    function withdrawMarketUsd(uint128 marketId, address target, uint256 amount) external;
```

Comment:
Can we make this change non breaking an always return zero. No signature change?

Reply:
This is a good catch, would we need to also update the market implementations otherwise? @sunnyvempati

Reply:
we don't use the return values in the perps market at least.  just fyi.

Reply:
Still al problem for all frontends as the signature changes after deployment all frontends will be dead until we update the signature. Please update.